CONTRIBUTING.md

```diff
@@ -58,33 +58,38 @@ Each new pull request should:
 
 ## Creating a new release
 
-### Minor/patch releases
-Minor and patch releases only need to change this repo. The Helm chart and deploy scripts
+### Patch releases
+Patch releases only need to change this repo. The Helm chart and deploy scripts
 will automatically pull in the latest changes.
 
-To deploy a minor or patch release, follow steps 2 and 3 from "Major releases" below.
+If the release involves changes to anything in the `deploy/` folder, it needs to be a minor
+or major release in order to prevent breaking the Helm chart.
 
-### Major releases
-Major releases need to change both this repository and the
+To deploy a patch release, follow steps 2 and 3 below.
```

Comment:
How would you feel about kicking DRY to the curb, and completely separating the instructions for `patch` vs `minor/major`? My gut feel, in running through this, is jumping from section to section and picking only the "parts you need" could lead to confusion. 

If someone wants to  update instructions 2&3 later in life, it may be unclear that they need to stand alone for the patch release process. Food for thought and happy to discuss.